Binary_search/Stress_test.py

```diff
@@ -0,0 +1,38 @@
+import random
+from Binary_search import Bin_search as binkS
+from Simple_search import Simple_search as simS
+
+n = 20
+K = 0
+N = [0]
+
+# Генерируем случайный набор массивов
+random.randint(0,3)

```

Comment:
You are not assigning this number to anything, so this line is useless :)
Let's remove it